src/tiledimage.js

```diff
@@ -1534,7 +1536,7 @@ function compareTiles( previousBest, tile ) {
 }
 
 function drawTiles( tiledImage, lastDrawn ) {
-    if (lastDrawn.length === 0) {
+    if (this.opacity === 0 || lastDrawn.length === 0) {
```

Comment:
This should be `tiledImage.opacity`, since drawTiles isn't actually a method of TiledImage.